test/CMakeLists.txt

```diff
@@ -73,6 +73,15 @@ if (UNIX AND ENABLE_SINGLE_FILES_WERROR)
     PROPERTIES COMPILE_FLAGS "-Wall -Werror")
 endif()
 
+if (APPLE)
+  find_package(OpenSSL REQUIRED)
+#  target_link_libraries(${test}  OpenSSL::SSL)
+#  target_link_libraries(${test} OpenSSL::Crypto)
+  target_link_libraries(valhalla_test OpenSSL::SSL)
+  target_link_libraries(valhalla_test OpenSSL::Crypto)
+endif()
```

Comment:
can you explain why we need to link this?

Reply:
already removed, i was thinking to use openssl to calculate file's hashsum to check if valhalla tiles are the same after applying elevation tiles (elevation tiles loaded from remote storage vs local storage). But this solution was changed to just checking files byte by byte.